packages/hadron-build/package.json

```diff
@@ -34,7 +34,7 @@
     "download": "^8.0.0",
     "electron": "^13.5.1",
     "electron-license": "^2.8.0",
-    "electron-packager": "^14.2.0",
+    "electron-packager": "^15.5.1",
```

Comment:
is this necessary? a while ago I  moved hadron build to `ts` and update this package and I recall that I had to change a few things, though I spotted those with the help of `ts` and maybe were already broken. Unless is a requirement I'd keep the old version and we can update `electron-packager` on its own, is not too easy to spot when it doesn't work properly, for example with those weird plugins like `ffmpeg`.

Reply:
Yes, darwin arm support was added only in 15. As we are running tests for every platform we support (except the darwin arm that I tried out manually) and none of the tests are failing, is it still a risk to bump it?

Reply:
It might, I don't think those were noticeable things, but no need to give up, we can take a look at the changes in the config too, i have the branch saved somewhere

Reply:
FWIW the only noticeable breaking change from v14 I saw when doing this update that is mentioned in the packager changelog is dropping node 10 support (which doesn't affect us), but obviously this doesn't mean nothing else changed, yeah. If you can find what you had to update that would be great